<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELSEBA Python Runner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PyRunResources.qrc">
    <normaloff>:/data/e.png</normaloff>:/data/e.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fntCombo">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>BatangChe</string>
        </property>
        <property name="fontFilters">
         <set>QFontComboBox::MonospacedFonts</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbFontSize">
        <property name="currentText">
         <string>5</string>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>36</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>72</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>418</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="40,60">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="vlOutputBtns">
            <item>
             <widget class="QPushButton" name="btnOutputOpen">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Open.png</normaloff>:/data/Icons/Open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnOutputSave">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Save.png</normaloff>:/data/Icons/Save.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnOutputClear">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Clear.png</normaloff>:/data/Icons/Clear.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="CodeEditor" name="txtOutput">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="plainText">
             <string>(output)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="vlInputBtns">
            <item>
             <widget class="QPushButton" name="btnInputOpen">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Open.png</normaloff>:/data/Icons/Open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnInputSave">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Save.png</normaloff>:/data/Icons/Save.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnInputClear">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="PyRunResources.qrc">
                <normaloff>:/data/Icons/Clear.png</normaloff>:/data/Icons/Clear.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="CodeEditor" name="txtInput">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="plainText">
             <string>ELSEBA Python Runner
Written by Bhathiya Perera

This Project Depends on
Qt5.3, Python
Python Syntax Highlight Code from Frankie Simon

(this is the input)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="vlCodeBtns">
          <item>
           <widget class="QPushButton" name="btnCodeOpen">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PyRunResources.qrc">
              <normaloff>:/data/Icons/Open.png</normaloff>:/data/Icons/Open.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCodeSave">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PyRunResources.qrc">
              <normaloff>:/data/Icons/Save.png</normaloff>:/data/Icons/Save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCodeClear">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PyRunResources.qrc">
              <normaloff>:/data/Icons/Clear.png</normaloff>:/data/Icons/Clear.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCodeDatabase">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PyRunResources.qrc">
              <normaloff>:/data/Icons/Snippets.png</normaloff>:/data/Icons/Snippets.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRun">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="PyRunResources.qrc">
              <normaloff>:/data/Icons/Run.png</normaloff>:/data/Icons/Run.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="CodeEditor" name="txtCode">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="plainText">
           <string>set_output(&quot;&quot;) # clear
dump(get_input())
write_output(&quot;append&quot;)
</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PyRunResources.qrc"/>
 </resources>
 <connections/>
</ui>
